src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj

```diff
@@ -14,15 +14,15 @@
     <Deterministic>true</Deterministic>
     <!-- Recommended: Embed symbols containing Source Link in the main file (exe/dll) -->
     <DebugType>embedded</DebugType>
-    <Version>0.1.1-preview.3</Version>
```

Comment:
If the package upgrades are not relevant to the fix, it might be a good idea to have a separate PR for that. Keeping a clean history (one fix per PR) makes cherry-picking and reverting changes much easier.

Reply:
Thanks @zengin, I will consider it in the next PRs.